be/src/util/system_metrics.cpp

```diff
@@ -241,13 +253,23 @@ void SystemMetrics::_update_cpu_metrics() {
 
 void SystemMetrics::_install_memory_metrics(MetricRegistry* registry) {
     _memory_metrics = std::make_unique<MemoryMetrics>();
+#ifndef USE_JEMALLOC
     registry->register_metric("memory_allocated_bytes", &_memory_metrics->allocated_bytes);
     registry->register_metric("total_thread_cache_bytes", &_memory_metrics->total_thread_cache_bytes);
     registry->register_metric("central_cache_free_bytes", &_memory_metrics->central_cache_free_bytes);
     registry->register_metric("transfer_cache_free_bytes", &_memory_metrics->transfer_cache_free_bytes);
     registry->register_metric("thread_cache_free_bytes", &_memory_metrics->thread_cache_free_bytes);
     registry->register_metric("pageheap_free_bytes", &_memory_metrics->pageheap_free_bytes);
     registry->register_metric("pageheap_unmapped_bytes", &_memory_metrics->pageheap_unmapped_bytes);
+#else
+    registry->register_metric("jemalloc_allocated_bytes", &_memory_metrics->jemalloc_allocated_bytes);
```

Comment:
Should the `active_bytes` be a common field, regarding of the memory allocator ?

Reply:
> Should the `active_bytes` be a common field, regarding of the memory allocator ?

I think it can, but the other metrics of tcmalloc and jemalloc are different, and it doesn't make much sense to keep a separate common field